docs/settings.rst

```diff
@@ -78,3 +78,14 @@ developing Page Objects.
 
 It's `recommended` to set this off into ``False`` by default since you might miss
 out on sporadic errors.
+
+
+SCRAPY_POET_TESTS_DIR
+---------------------
+
+Default: ``None``
+
+Sets the location where the ``create_po_test`` command creates tests. When set
+to ``None``, uses the ``fixtures`` directory in the current directory.
```

Comment:
We use `settings.get("SCRAPY_POET_TESTS_DIR", "fixtures")`, does this mean that the default is `"fixtures"`? I don't think so, but I don't think we can make a non-None default here?

Reply:
My point is we don't have default values for settings (non-None value returned by get() when the user didn't set it) but in this case we handle unset values as set to "fixtures". 

It's different from having a true default value of "fixtures" because you can't change it to None in your settings.py.

Reply:
You mean that setting `SCRAPY_POET_TESTS_DIR=None` or `SCRAPY_POET_TESTS_DIR=""` in `settings.py` would break this? If so, we could use `settings.get("SCRAPY_POET_TESTS_DIR") or "fixtures"` instead.

Reply:
Well, my initial question was only "how to document this properly", as replacing the current doc with just "Default: "fixtures"" sounds nice but will be technically wrong (as, yes, setting SCRAPY_POET_TESTS_DIR=None won't change the setting).

Reply:
How about setting `Default: "fixtures"` in the docs and when `SCRAPY_POET_TESTS_DIR=None`, the command fails?

Reply:
@BurnzZ I don't think we can see if it's None because the user passed None or because nothing was set?

Reply:
I see, I totally missed that. In that case, I think documenting it with `Default: None` would be alright.